src/components/enter-phone-number/enter-phone-number-controller.ts

```diff
@@ -84,7 +84,9 @@ export function enterPhoneNumberPost(
             req.query.actionType as SecurityCodeErrorType
           ),
           support2hrLockout: support2hrLockout(),
-          isAccountCreationJourney: true,
+          isAccountCreationJourney:
+            req.session.user.isAccountCreationJourney ||
+            req.session.user.isAccountPartCreated,
```

Comment:
This looks like a separate change / fix to me - is that right? If so, could we either put it in its own commit with tests and explanation of what it does, or add something to the description of this commit that explains why this was changed? In the commit message for this commit you say "The enter-phone-controller sent the correct journey type", but this change implies otherwise :-)

Reply:
Will do, agreed